dags/requirements.txt

```diff
@@ -0,0 +1 @@
+/usr/local/airflow/plugins/airflow-0.0.0-py3-none-any.whl
```

Comment:
Do you really want a full path to a wheel file?

Reply:
I do. This is purely for local development and unique to the MWAA environment. Developers will be providing that wheel in the plugins directory before building/starting the image.